xdsl/interactive/app.py

```diff
@@ -135,6 +135,10 @@ class InputApp(App[None]):
     DataTable displaying the diff of operation names and counts of the input and output
     text areas.
     """
+    current_argument_pass: tuple[tuple[type[ModulePass], PipelinePassSpec], ...] = ()
+    """
+    Saves a tuple containing the constructed ModulePass, PipelinePassSpec tuple of a pass with arguments requiring user input.
+    """
```

Comment:
I thought we had agreed to remove this, I'm still not sure what breaks if this variable is removed. It feels scary to me, complexity-wise, to add this variable without a well-defined meaning. I don't know what a "current argument pass" is.

Reply:
I removed ability to expand argument passes. What do you think now?